test/test_queue.js

```diff
@@ -1915,7 +1920,10 @@ describe('Queue', () => {
       queue.add({});
       queue.add({});
 
-      queue.on('completed', _.after(2, () => done()));
+      queue.on(
```

Comment:
running `npm run prettier` formatted this line, I can back out the change if you like.